### ISSUE TITLE
Use sass --embedded once it's released

### ISSUE THREAD

**Issue by AesaraB** [gohugoio/hugo#11059]
Hi, the [dart-sass-embedded](https://github.com/sass/dart-sass-embedded) compiler has been merged into dart-sass. Please consider using `sass --embedded` instead of ``dart-sass-embedded`` for forwards compatibility.

**Comment by bep** [comment]
Can you point to the released version of this `sass` binary?

**Comment by tianheg** [comment]
> The embedded compiler has been merged into [the primary Dart Sass repository](https://github.com/sass/dart-sass), and further releases will be included [as part of Dart Sass itself](https://github.com/sass/dart-sass#embedded-dart-sass).

The new version of `sass` haven't been released yet

**Comment by AesaraB** [comment]
> > The embedded compiler has been merged into [the primary Dart Sass repository](https://github.com/sass/dart-sass), and further releases will be included [as part of Dart Sass itself](https://github.com/sass/dart-sass#embedded-dart-sass).
> > -- readme of sass/dart-sass-embedded
> 
> The new version of `sass` haven't been released yet

Apologies, I didn't look into this enough. I'll add a comment updating this issue when the next version of dart-sass is released.

**Comment by bep** [comment]
Thanks, this is already on our radar, but need a Dart Sass release before we can implement this.

**Comment by bep** [comment]
@jmooring yea, well, this will all start out in

https://github.com/bep/godartsass

Which has some basic benchmarks which will hopefully give raise a red flag if the performance has taken a hit.

Note that they also plan a change in the wire format (I'm not sure if that's implemented) that needs some update in the godartsass library to work -- as I understood it, that was motivated by performance.

**Comment by bep** [comment]
Here's the changes I mentioned above: https://github.com/sass/dart-sass/issues/1980

**Comment by AesaraB** [comment]
The latest version of Dart Sass has released with the inclusion of ``sass --embedded``, please see [Dart Sass 1.63.0](https://github.com/sass/dart-sass/releases/tag/1.63.0)

**Comment by bep** [comment]
I have upgraded the upstream library to v2 of the protocol, but I need to understand this issue

https://github.com/bep/godartsass/issues/19

Before I can pull it into Hugo.

**Comment by XhmikosR** [comment]
Would it make sense to look into `./node_modules` for `sass` now that the regular Dart Sass binary is supported? Similar to postcss?

**Comment by bep** [comment]
@XhmikosR maybe, but closed issues/PRs are closed and I almost never read comments in closed threads. I suggest you create a new issue.
